climateconnect/climateconnect
# Fix: Skills/Connections on edit project don't wrap

## Description
Closes #986
Added a flex-wrap to skills/connections so they dont float off the page, added a margin between the chips.

Before:
![Screenshot from 2022-11-11 15-30-35](https://user-images.githubusercontent.com/49129464/201361789-5c066a64-4d9e-4527-9a2b-23448d060190.png)
After:
![Screenshot from 2022-11-11 15-31-25](https://user-images.githubusercontent.com/49129464/201361738-b98d65b2-5a0f-49cb-a68b-cfb5db8bfd4e.png)

## Test plan

1. Go to a project for which you have editing rights
2. Click edit project
3. Scroll down to add skills/add connections
4. Add many skills/connections to see the wrapping take place

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
